docs/components/jupyter_dask_cloud_to_cluster.md

```diff
@@ -0,0 +1,7 @@
+#
```

Comment:
This top "#" character blocks the titles from the original repository to be displayed - If I remove it, the title from the repository README.md  (e.g. "JupyterDask on SLURM") is properly displayed. 

Or is there some other reason why this should be here?

```suggestion

```

Reply:
The reason is that the "title" is displayed in the left panel. If `#` is removed, the tile is again shown on the top of the page.  I think displaying the title two times makes the page look busy, see the screenshot below. WHat do you think? :thinking: 
![Screenshot from 2024-06-28 15-11-10](https://github.com/RS-DAT/RS-DAT/assets/55081872/3b40b3df-4fcd-4a7a-b674-9f35ac51d7a2)

Reply:
Oh, I actually like it with the title on both sides, it looks clearer to me which tab I am actually looking at. But if you like it better without the title, feel free to remove it :)